api/v2/sdks/client-side-identity.md

```diff
@@ -180,34 +204,34 @@ The [callback function](#callback-function) returns the `status` field values as
 | Status | Advertising Token Availability | Description |
 | :--- | :--- | :--- |
 | `ESTABLISHED` | Available | The identity is valid, was set from the passed value or the first-party cookie, and is now available for targeted advertising. |
-| `REFRESHED` | Available | The identity was successfully refreshed by a call to the UID2 operator is now available for targeted advertising. |
+| `REFRESHED` | Available | The identity was successfully refreshed by a call to the UID2 operator, and is now available for targeted advertising. |
 | `EXPIRED` | Not available | No identity is available for targeted advertising, as the SDK failed to refresh the token. Since there is still a valid refresh token available, auto-refresh attempts will continue. |
-| `REFRESH_EXPIRED` | Not available | No identity is available for targeted advertising, as the refresh token on the first-party cookie or the passed identity has expired.  |
-| `NO_IDENTITY` | Not available | No identity is available for targeted advertising, as a first-party cookie was not set and no identity has been passed to the `init()` function.  |
-| `INVALID` | Not available | No identity is available for targeted advertising, as the SDK failed to parse the first-party cookie or the passed identity. |
-| `OPTOUT` | Not available | No identity is available for targeted advertising, as the user has opted out from refreshing identity. |
+| `REFRESH_EXPIRED` | Not available | No identity is available for targeted advertising, because the refresh token on the first-party cookie or the passed identity has expired.  |
+| `NO_IDENTITY` | Not available | No identity is available for targeted advertising, because a first-party cookie was not set and no identity has been passed to the `init()` function.  |
+| `INVALID` | Not available | No identity is available for targeted advertising, because the SDK failed to parse the first-party cookie or the passed identity. |
+| `OPTOUT` | Not available | No identity is available for targeted advertising, because the user has opted out and therefore the identity cannot be refreshed. |
 
 If the identity is not available, to determine the best course of action, use the [isLoginRequired()](#isloginrequired-boolean) function.
 
 ### getAdvertisingToken(): string
 
 Gets the current advertising token. 
 
-Be sure to call this function *after* calling  [init()](#initopts-object-void) and invoking the supplied callback, for example: 
+Before calling this function, be sure to call [init()](#initopts-object-void) and invoke the supplied callback, as shown in the following example. 
 
 ```html
 <script>
   let advertisingToken = __uid2.getAdvertisingToken();
```

Comment:
Unfortunately, this example doesn't do the thing it's meant to be an example of (calling init and waiting until the callback has been invoked). We'll need a more complete example :(

Reply:
Noted for follow-up on APIDOCS-1176. Thx!